libnmstate/nm/dns.py

```diff
@@ -14,21 +14,30 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
+import six
+
 from itertools import chain
 from operator import itemgetter
 
 from libnmstate import iplib
 from libnmstate.error import NmstateInternalError
 from libnmstate.nm import nmclient
 from libnmstate.nm import active_connection as nm_ac
+from libnmstate.nm.route import IPV4_DEFAULT_GATEWAY_DESTINATION
+from libnmstate.nm.route import IPV6_DEFAULT_GATEWAY_DESTINATION
 from libnmstate.schema import DNS
+from libnmstate.schema import Interface
+from libnmstate.schema import Route
 
 
 DNS_DEFAULT_PRIORITY_VPN = 50
 DNS_DEFAULT_PRIORITY_OTHER = 100
 DNS_METADATA = '_dns'
 DNS_METADATA_PRIORITY = '_priority'
 DEFAULT_DNS_PRIORITY = 0
+# The 40 is chose as default DHCP DNS priority of VPN is 50, the static
```

Comment:
The English here is a bit problematic.
Maybe: `The value 40 has been chosen to prioritize static entries over DHCP DNS (#??) and VPN(50).`

Reply:
Choose 40 to prioritize...